### ISSUE TITLE
Question: Supported response types

### ISSUE THREAD

**Issue by kyvour** [panva/node-oidc-provider#246]
Hi

I have a small question about supported response times. 
As far as I understand since the library fully implements OpenID Connect Core 1.0, all listed in [Appendix A](https://openid.net/specs/openid-connect-core-1_0.html#AuthorizationExamples) response types are supported:
- response_type=code
- response_type=id_token
- response_type=id_token token
- response_type=code id_token
- response_type=code token
- response_type=code id_token token

Am I understand right?

**Comment by panva** [comment]
yes, and you can [configure](https://github.com/panva/node-oidc-provider/blob/v2.17.0/docs/configuration.md#responsetypes) your provider to only support a subset.
